src/react_native/image_crop_picker.cljs

```diff
@@ -0,0 +1,27 @@
+(ns react-native.image-crop-picker
+  (:require ["react-native-image-crop-picker" :default image-picker]))
+
+(defn show-access-error
+  [o]
+  (js/console.log (.-message ^js o)))
+
+(defn show-image-picker
+  ([callback]
+   (show-image-picker callback nil))
+  ([callback
+    {:keys [media-type]
+     :or   {media-type "any"}
+     :as   props}]
+   (-> ^js image-picker
+       (.openPicker (clj->js (merge {:mediaType media-type} props)))
+       (.then #(callback (.-path ^js %)))
+       (.catch show-access-error))))
+
+(defn show-image-picker-camera
+  ([callback]
+   (show-image-picker-camera callback nil))
+  ([callback props]
+   (-> ^js image-picker
+       (.openCamera (clj->js props))
+       (.then #(callback (.-path ^js %)))
+       (.catch show-access-error))))
```

Comment:
Would be great to expose the promises without both the callback and error handling (`show-access-error`), so that the resolution/rejection can be handled downstream and potentially chained with other promises (e.g. inside an effect).

Reply:
should we create an issue?